investment-app/TRINV.Api/TRINV.Infrastructure/Configurations/DependencyInjection.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TRINV.Infrastructure.Configurations;
+
+using Interfaces;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+public static class DependencyInjection
```

Comment:
Maybe if you want you can name it . DI -> Which means Dependency Injection

Reply:
No worries. 
I told you earlier today that you don't have to register it in Program.cs but that turns out to not be true. 
But still it will help to keep Program.cs cleaner.
As you need one line of code to register all services for a layer.

Reply:
Done.